<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="HttpException.xml" source-language="en-US" target-language="es-ES">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac51864e8fe52f0eb87b7c23fbfe505d15cf9681ede.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1864e8fe52f0eb87b7c23fbfe505d15cf9681ede</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Web.HttpException">
          <source>Describes an exception that occurred during the processing of HTTP requests.</source>
          <target state="translated">Describe una excepción que apareció durante el procesamiento de solicitudes HTTP.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> class is an HTTP-specific exception class that enables ASP.NET to generate exception information.</source>
          <target state="translated">La <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> clase es una clase de excepción específica de HTTP que permite a ASP.NET generar información de excepción.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>For more information about throwing and handling exceptions, see <bpt id="p1">[</bpt>Exceptions<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>.</source>
          <target state="translated">Para obtener más información acerca de cómo iniciar y controlar las excepciones, vea <bpt id="p1">[</bpt>excepciones<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The following code example demonstrates how to use the <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> class to raise customized exceptions.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra cómo utilizar la <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> clase para generar excepciones personalizadas.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>The <ph id="ph1">`Button_Click`</ph> method determines which button was clicked, and then calls either the <ph id="ph2">`CheckNumber`</ph> or <ph id="ph3">`CheckBoolean`</ph> method.</source>
          <target state="translated">El <ph id="ph1">`Button_Click`</ph> método determina qué botón se hizo clic y, a continuación, llama a la <ph id="ph2">`CheckNumber`</ph> o <ph id="ph3">`CheckBoolean`</ph> método.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>If the user-entered value in the text box does not correspond to the expected type, an <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> exception is created and thrown with a customized message based on the error type.</source>
          <target state="translated">Si el valor escrito por el usuario en el cuadro de texto no se corresponde con el tipo esperado, un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> excepción se crea y se produce un mensaje personalizado en función del tipo de error.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">Este ejemplo tiene un cuadro de texto que acepta datos proporcionados por el usuario, lo que puede suponer una amenaza para la seguridad.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">De forma predeterminada, ASP.NET Web Pages valida que los datos proporcionados por el usuario no incluyen elementos HTML ni de script.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpException">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Para más información, consulte <bpt id="p1">[</bpt>Información general sobre los ataques mediante scripts<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="T:System.Web.HttpException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class and creates an empty <ph id="ph2">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> object.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> y crea un objeto <ph id="ph2">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> vacío.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">Al controlar excepciones, puede ser útil capturar una serie de excepciones relacionadas con la excepción externa que se producen en respuesta a una excepción interna.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">Una referencia a la excepción interna que produjo la excepción externa está disponible en la <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> propiedades de la excepción externa.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">Este mecanismo conserva la información de error que se llevan a las excepciones anteriores, incluidas las excepciones originales, al tiempo que permite crear excepciones externas más significativas.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Para obtener más información, vea <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra la <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor de la <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> clase.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value through a text box and checks whether it is an integer.</source>
          <target state="translated">El <ph id="ph1">`CheckNumber`</ph> método acepta un valor escrito por el usuario a través de un cuadro de texto y comprueba si es un entero.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>If the value is not an integer, an exception is thrown, and then a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">Si el valor no es un entero, se produce una excepción y, a continuación, un nuevo <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> objeto se crea y se produce.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message is displayed on the browser.</source>
          <target state="translated">Detecta la excepción en el <ph id="ph1">`Button_Click`</ph> controlador de eventos y el mensaje de error se muestra en el explorador.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">Este ejemplo tiene un cuadro de texto que acepta datos proporcionados por el usuario, lo que puede suponer una amenaza para la seguridad.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">De forma predeterminada, ASP.NET Web Pages valida que los datos proporcionados por el usuario no incluyen elementos HTML ni de script.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Para más información, consulte <bpt id="p1">[</bpt>Información general sobre los ataques mediante scripts<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Mensaje de error que se muestra al cliente cuando se produce la excepción.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using a supplied error message.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> utilizando el mensaje de error especificado.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra la <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor de la <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> clase.</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>An <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> is thrown if a user-entered value is 0.</source>
          <target state="translated">Un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> se produce si un valor escrito por el usuario es 0.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">Este ejemplo tiene un cuadro de texto que acepta datos proporcionados por el usuario, lo que puede suponer una amenaza para la seguridad.</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">De forma predeterminada, ASP.NET Web Pages valida que los datos proporcionados por el usuario no incluyen elementos HTML ni de script.</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Para más información, consulte <bpt id="p1">[</bpt>Información general sobre los ataques mediante scripts<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>The HTTP response status code sent to the client corresponding to this error.</source>
          <target state="translated">Código de estado de respuesta HTTP enviado al cliente que corresponde a este error.</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Mensaje de error que se muestra al cliente cuando se produce la excepción.</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code and an error message.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> utilizando un código de estado de respuesta HTTP y un mensaje de error.</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">
          <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> que contiene los datos serializados del objeto que hacen referencia a la excepción que se va a producir.</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that holds the contextual information about the source or destination.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> que contiene la información contextual sobre el origen o el destino.</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class with serialized data.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> con datos serializados.</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor is called during deserialization to reconstitute the exception object that is transmitted over a stream.</source>
          <target state="translated">El <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor se llama durante la deserialización para reconstituir el objeto de excepción transmitido en una secuencia.</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>For more information, see <bpt id="p1">[</bpt>XML and SOAP Serialization<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</source>
          <target state="translated">Para obtener más información, consulte <bpt id="p1">[</bpt>serialización XML y SOAP<ept id="p1">](~/docs/standard/serialization/xml-and-soap-serialization.md)</ept>.</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Mensaje de error que se muestra al cliente cuando se produce la excepción.</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, if any, that threw the current exception.</source>
          <target state="translated">Propiedad <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, si hay alguna, que ha producido la excepción actual.</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an error message and the <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> property.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> utilizando un mensaje de error y la propiedad <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">Al controlar excepciones, puede ser útil capturar una serie de excepciones relacionadas con la excepción externa que se producen en respuesta a una excepción interna.</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">Una referencia a la excepción interna que produjo la excepción externa está disponible en la <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> propiedades de la excepción externa.</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">Este mecanismo conserva la información de error que se llevan a las excepciones anteriores, incluidas las excepciones originales, al tiempo que permite crear excepciones externas más significativas.</target>       </trans-unit>
        <trans-unit id="144" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Para obtener más información, vea <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="145" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra la <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor de la <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> clase.</target>       </trans-unit>
        <trans-unit id="146" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value through a text box and checks whether it is an integer.</source>
          <target state="translated">El <ph id="ph1">`CheckNumber`</ph> método acepta un valor escrito por el usuario a través de un cuadro de texto y comprueba si es un entero.</target>       </trans-unit>
        <trans-unit id="147" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>If the value is not an integer, an exception is thrown, and then in the catch block, a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">Si el valor no es un entero, se produce una excepción y, a continuación, en el bloque catch, un nuevo <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> objeto se crea y se produce.</target>       </trans-unit>
        <trans-unit id="148" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message is displayed on the browser.</source>
          <target state="translated">Detecta la excepción en el <ph id="ph1">`Button_Click`</ph> controlador de eventos y el mensaje de error se muestra en el explorador.</target>       </trans-unit>
        <trans-unit id="149" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">Este ejemplo tiene un cuadro de texto que acepta datos proporcionados por el usuario, lo que puede suponer una amenaza para la seguridad.</target>       </trans-unit>
        <trans-unit id="150" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">De forma predeterminada, ASP.NET Web Pages valida que los datos proporcionados por el usuario no incluyen elementos HTML ni de script.</target>       </trans-unit>
        <trans-unit id="151" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Exception)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Para más información, consulte <bpt id="p1">[</bpt>Información general sobre los ataques mediante scripts<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="152" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Mensaje de error que se muestra al cliente cuando se produce la excepción.</target>       </trans-unit>
        <trans-unit id="153" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The exception code that defines the error.</source>
          <target state="translated">Código de excepción que define el error.</target>       </trans-unit>
        <trans-unit id="154" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an error message and an exception code.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> utilizando un mensaje de error y un código de excepción.</target>       </trans-unit>
        <trans-unit id="155" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra la <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor de la <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> clase.</target>       </trans-unit>
        <trans-unit id="156" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>An <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> exception is thrown if a user-entered value is 0.</source>
          <target state="translated">Un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> excepción se produce si un valor escrito por el usuario es 0.</target>       </trans-unit>
        <trans-unit id="157" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">Este ejemplo tiene un cuadro de texto que acepta datos proporcionados por el usuario, lo que puede suponer una amenaza para la seguridad.</target>       </trans-unit>
        <trans-unit id="158" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">De forma predeterminada, ASP.NET Web Pages valida que los datos proporcionados por el usuario no incluyen elementos HTML ni de script.</target>       </trans-unit>
        <trans-unit id="159" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.String,System.Int32)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Para más información, consulte <bpt id="p1">[</bpt>Información general sobre los ataques mediante scripts<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="160" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The HTTP response status code displayed on the client.</source>
          <target state="translated">Código de estado de respuesta HTTP que se muestra en el cliente.</target>       </trans-unit>
        <trans-unit id="161" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Mensaje de error que se muestra al cliente cuando se produce la excepción.</target>       </trans-unit>
        <trans-unit id="162" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, if any, that threw the current exception.</source>
          <target state="translated">Propiedad <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>, si hay alguna, que ha producido la excepción actual.</target>       </trans-unit>
        <trans-unit id="163" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code, an error message, and the <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> property.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> utilizando un código de estado de respuesta HTTP, un mensaje de error y la propiedad <ph id="ph2">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="164" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>When handling exceptions, it can be useful to capture a series of related exceptions with the outer exception being thrown in response to an inner exception.</source>
          <target state="translated">Al controlar excepciones, puede ser útil capturar una serie de excepciones relacionadas con la excepción externa que se producen en respuesta a una excepción interna.</target>       </trans-unit>
        <trans-unit id="165" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>A reference to the inner exception that caused the outer exception is available from the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property of the outer exception.</source>
          <target state="translated">Una referencia a la excepción interna que produjo la excepción externa está disponible en la <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> propiedades de la excepción externa.</target>       </trans-unit>
        <trans-unit id="166" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>This mechanism preserves the error information that is carried by earlier exceptions, including the original exceptions, while allowing you to create more meaningful outer exceptions.</source>
          <target state="translated">Este mecanismo conserva la información de error que se llevan a las excepciones anteriores, incluidas las excepciones originales, al tiempo que permite crear excepciones externas más significativas.</target>       </trans-unit>
        <trans-unit id="167" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>For more information, see <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</source>
          <target state="translated">Para obtener más información, vea <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="168" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra la <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor de la <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> clase.</target>       </trans-unit>
        <trans-unit id="169" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>The <ph id="ph1">`CheckNumber`</ph> method accepts a user-entered value and checks whether it is an integer.</source>
          <target state="translated">El <ph id="ph1">`CheckNumber`</ph> método acepta un valor escrito por el usuario y comprueba si es un entero.</target>       </trans-unit>
        <trans-unit id="170" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>If the value is not an integer, an exception is thrown, and then a new <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object containing the HTTP response status code, the exception's message, and any inner exception is created.</source>
          <target state="translated">Si el valor no es un entero, se produce una excepción y, a continuación, un nuevo <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> se crea el objeto que contiene el código de estado de respuesta HTTP, mensaje de excepción y cualquier excepción interna.</target>       </trans-unit>
        <trans-unit id="171" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>That exception is caught in the <ph id="ph1">`Button_Click`</ph> event handler and the error message, error code, and inner exception are displayed.</source>
          <target state="translated">Detecta la excepción en el <ph id="ph1">`Button_Click`</ph> se muestran el controlador de eventos y el mensaje de error, el código de error y la excepción interna.</target>       </trans-unit>
        <trans-unit id="172" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">Este ejemplo tiene un cuadro de texto que acepta datos proporcionados por el usuario, lo que puede suponer una amenaza para la seguridad.</target>       </trans-unit>
        <trans-unit id="173" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">De forma predeterminada, ASP.NET Web Pages valida que los datos proporcionados por el usuario no incluyen elementos HTML ni de script.</target>       </trans-unit>
        <trans-unit id="174" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Exception)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Para más información, consulte <bpt id="p1">[</bpt>Información general sobre los ataques mediante scripts<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="175" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The HTTP response status code displayed on the client.</source>
          <target state="translated">Código de estado de respuesta HTTP que se muestra en el cliente.</target>       </trans-unit>
        <trans-unit id="176" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Mensaje de error que se muestra al cliente cuando se produce la excepción.</target>       </trans-unit>
        <trans-unit id="177" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The exception code that defines the error.</source>
          <target state="translated">Código de excepción que define el error.</target>       </trans-unit>
        <trans-unit id="178" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> class using an HTTP response status code, an error message, and an exception code.</source>
          <target state="translated">Inicializa una nueva instancia de la clase <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> utilizando un código de estado de respuesta HTTP, un mensaje de error y un código de excepción.</target>       </trans-unit>
        <trans-unit id="179" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The following code example demonstrates the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor of the <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> class.</source>
          <target state="translated">En el ejemplo de código siguiente se muestra la <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor de la <ph id="ph2">&lt;xref:System.Web.HttpException&gt;</ph> clase.</target>       </trans-unit>
        <trans-unit id="180" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The user name and email information are entered by the user in the provided text boxes.</source>
          <target state="translated">La información de nombre y el correo electrónico de usuario son especificados por el usuario en los cuadros de texto proporcionado.</target>       </trans-unit>
        <trans-unit id="181" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>If any of the text boxes are left blank, an <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> object is created and thrown.</source>
          <target state="translated">Si cualquiera de los cuadros de texto se deja en blanco, un <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> objeto se crea y se produce.</target>       </trans-unit>
        <trans-unit id="182" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>The error code of the <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> is obtained by the <ph id="ph2">&lt;xref:System.Web.HttpException.GetHttpCode%2A&gt;</ph> method and displayed on the Web page.</source>
          <target state="translated">El código de error de la <ph id="ph1">&lt;xref:System.Web.HttpException&gt;</ph> se obtiene mediante la <ph id="ph2">&lt;xref:System.Web.HttpException.GetHttpCode%2A&gt;</ph> método y se muestran en la página Web.</target>       </trans-unit>
        <trans-unit id="183" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>This example has a text box that accepts user input, which is a potential security threat.</source>
          <target state="translated">Este ejemplo tiene un cuadro de texto que acepta datos proporcionados por el usuario, lo que puede suponer una amenaza para la seguridad.</target>       </trans-unit>
        <trans-unit id="184" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>By default, ASP.NET Web pages validate that user input does not include script or HTML elements.</source>
          <target state="translated">De forma predeterminada, ASP.NET Web Pages valida que los datos proporcionados por el usuario no incluyen elementos HTML ni de script.</target>       </trans-unit>
        <trans-unit id="185" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.#ctor(System.Int32,System.String,System.Int32)">
          <source>For more information, see <bpt id="p1">[</bpt>Script Exploits Overview<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</source>
          <target state="translated">Para más información, consulte <bpt id="p1">[</bpt>Información general sobre los ataques mediante scripts<ept id="p1">](http://msdn.microsoft.com/library/772c7312-211a-4eb3-8d6e-eec0aa1dcc07)</ept>.</target>       </trans-unit>
        <trans-unit id="186" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>The error message displayed to the client when the exception is thrown.</source>
          <target state="translated">Mensaje de error que se muestra al cliente cuando se produce la excepción.</target>       </trans-unit>
        <trans-unit id="187" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>Creates a new <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> exception based on the error code that is returned from the Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> method.</source>
          <target state="translated">Crea una nueva excepción <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> basada en el código de error devuelto por el método <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> de la API Win32.</target>       </trans-unit>
        <trans-unit id="188" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.CreateFromLastError(System.String)">
          <source>An <ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> based on the error code that is returned from a call to the Win32 API <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> method.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Web.HttpException" /&gt;</ph> basada en el código de error que se devuelve a partir de una llamada al método <ph id="ph2">&lt;see langword="GetLastError()" /&gt;</ph> de la API Win32.</target>       </trans-unit>
        <trans-unit id="189" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHtmlErrorMessage">
          <source>Gets the HTML error message to return to the client.</source>
          <target state="translated">Obtiene el mensaje de error HTML que se va a devolver al cliente.</target>       </trans-unit>
        <trans-unit id="190" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHtmlErrorMessage">
          <source>The HTML error message.</source>
          <target state="translated">Mensaje de error HTML.</target>       </trans-unit>
        <trans-unit id="191" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHttpCode">
          <source>Gets the HTTP response status code to return to the client.</source>
          <target state="translated">Obtiene el código de estado de respuesta HTTP que se va a devolver al cliente.</target>       </trans-unit>
        <trans-unit id="192" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetHttpCode">
          <source>A non-zero HTTP code representing the exception or the <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph> code; otherwise, HTTP response status code 500.</source>
          <target state="translated">Es el código HTTP distinto de cero que representa la excepción o el código <ph id="ph1">&lt;see cref="P:System.Exception.InnerException" /&gt;</ph>; de lo contrario, es el código de estado de respuesta HTTP 500.</target>       </trans-unit>
        <trans-unit id="193" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>If there is a non-zero HTTP code, it is returned.</source>
          <target state="translated">Si hay un código HTTP distinto de cero, se devuelve.</target>       </trans-unit>
        <trans-unit id="194" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>Otherwise, the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> code is returned.</source>
          <target state="translated">En caso contrario, el <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> se devuelve el código.</target>       </trans-unit>
        <trans-unit id="195" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>If neither an <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> code nor a non-zero HTTP code is available, the HTTP response status code 500 is returned.</source>
          <target state="translated">Si no un <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> hay código ni un código HTTP distinto de cero, se devuelve el código de estado de respuesta HTTP 500.</target>       </trans-unit>
        <trans-unit id="196" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetHttpCode">
          <source>For more information on HTTP response status codes, see the Status Code Definitions of the Hypertext Transfer Protocol - HTTP/1.1 on the World <bpt id="p1">[</bpt>Wide Web Consortium (W3C) Web site<ept id="p1">](http://go.microsoft.com/fwlink/?linkid=37125)</ept>.</source>
          <target state="translated">Para obtener más información sobre códigos de estado de respuesta HTTP, vea las definiciones de código de estado del protocolo de transferencia de hipertexto - HTTP/1.1 en el mundo <bpt id="p1">[</bpt>sitio Web de Wide Web Consortium (W3C)<ept id="p1">](http://go.microsoft.com/fwlink/?linkid=37125)</ept>.</target>       </trans-unit>
        <trans-unit id="197" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">
          <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> que contiene los datos serializados del objeto que hacen referencia a la excepción que se va a producir.</target>       </trans-unit>
        <trans-unit id="198" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that holds the contextual information about the source or destination.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> que contiene la información contextual sobre el origen o el destino.</target>       </trans-unit>
        <trans-unit id="199" translate="yes" xml:space="preserve" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Gets information about the exception and adds it to the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object.</source>
          <target state="translated">Obtiene información sobre la excepción y la agrega al objeto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="200" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpException.GetObjectData%2A&gt;</ph> method adds information about the exception to the <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> object that is targeted for serialization.</source>
          <target state="translated">El <ph id="ph1">&lt;xref:System.Web.HttpException.GetObjectData%2A&gt;</ph> método agrega información sobre la excepción a la <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> objeto destinado para la serialización.</target>       </trans-unit>
        <trans-unit id="201" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>During deserialization, the exception is reconstituted from the <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> that is transmitted over the stream.</source>
          <target state="translated">Durante la deserialización, la excepción se reconstituye a partir del <ph id="ph1">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> que se transmiten a través de la secuencia.</target>       </trans-unit>
        <trans-unit id="202" translate="yes" xml:space="preserve" uid="P:System.Web.HttpException.WebEventCode">
          <source>Gets the event codes that are associated with the HTTP exception.</source>
          <target state="translated">Obtiene los códigos de evento asociados a la excepción HTTP.</target>       </trans-unit>
        <trans-unit id="203" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpException.WebEventCode">
          <source>An integer representing a Web event code.</source>
          <target state="translated">Entero que representa un código de evento web.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>